### PR TITLE
Icon compatibility with Font Awesome 5

### PR DESCRIPTION
**Pull request by radekdostal**
Unified icon names to be compatible with FA 5.

### CODE REVIEW

File: src/templates/datagrid.latte [hunk 1]
```diff
@@ -135,7 +135,7 @@
 												<a n:href="showAllColumns!" class="ajax"><i n:block = "icon-eye" class="{$icon_prefix}eye"></i> {='ublaboo_datagrid.show_all_columns'|translate}</a>
 											</li>
 											<li n:if="$control->hasSomeColumnDefaultHide()">
-												<a n:href="showDefaultColumns!" class="ajax"><i n:block = "icon-repeat" class="{$icon_prefix}repeat"></i> {='ublaboo_datagrid.show_default_columns'|translate}</a>
+												<a n:href="showDefaultColumns!" class="ajax"><i n:block = "icon-repeat" class="{$icon_prefix}redo"></i> {='ublaboo_datagrid.show_default_columns'|translate}</a>
```
Comment (paveljanda) [inline src/templates/datagrid.latte]: That one (and only one) would break backward compatibility.

Comment (radekdostal) [reply src/templates/datagrid.latte]: "redo" should be an alias for "repeat". Same as "times" and "remove".
See https://fontawesome.com/v4.7.0/icons/ and search "redo".

Comment (paveljanda) [reply src/templates/datagrid.latte]: `redo` is not an icon in `v4`. Try it by yourself. :)

Comment (radekdostal) [reply src/templates/datagrid.latte]: Hm. You are right.
There are more incompatible icons (FA4 -> FA5 - Free):
- repeat -> redo
- arrows -> arrows-alt
- check-square-o -> check-square (filled)
- square-o -> square (filled)

I don't know, how to support it. Add properties like DataGrid::$icon_prefix?

Comment (kleinpetr) [reply src/templates/datagrid.latte]: next one different icon is
gear -> cog

Comment (paveljanda) [reply src/templates/datagrid.latte]: `fa-repeat` souns quiet OK to  me. :)

**Comment by f3l1x** [comment]
This is so old branch, can we close it?

**Comment by paveljanda** [comment]
@f3l1x jo

**Pull request closed without merging**